astropy/samp/hub_script.py

```diff
@@ -10,8 +10,6 @@
 
 from .hub import SAMPHubServer
 
-__all__ = ['main']
```

Comment:
I think maybe we should still have ``__all__`` but just put ``hub_script`` in it instead - I think I must have renamed main -> hub_script and forgot to update it in ``__all__``.

Reply:
[`__all__` is used for * imports.](
https://docs.python.org/3/tutorial/modules.html#importing-from-a-packag)  and my sense is that no one has been doing __from hub_script import *__ because * imports are less popular today that they used to be in the past.

Making the suggested change.